### ISSUE TITLE
Uncaught TypeError: Cannot read property 'previousSibling' of undefined 

### ISSUE THREAD

**Issue by ElijahLynn** [unicorn-fail/dreditor#223]
On trying to make a comment in the diff viewer, saving and then clicking paste I get an error of:

`Uncaught TypeError: Cannot read property 'previousSibling' of undefined`

Dreditor 1.2.10 on Chrome 36 on Ubuntu 14.04

**Comment by weitzman** [comment]
I get this too. I'm using Dreditor 1.2.10, Chrome Version 36.0.1985.14, OSX 10.9.

Backtrace:

```
    Uncaught TypeError: Cannot read property 'previousSibling' of undefined VM458:1929

(anonymous function) VM458:1929
b.extend.each js_xAPl0qIk9eowy_iS9tNkCWXLUVoat94SQT48UBCFkyQ.js:34
Drupal.dreditor.patchReview.paste VM458:1911
(anonymous function) VM458:2410
c.event.handle js_xAPl0qIk9eowy_iS9tNkCWXLUVoat94SQT48UBCFkyQ.js:64
h.handle.o
```

**Comment by weitzman** [comment]
I'll add this is kinda critical. If you can't paste, you have just lost your review (unless you are willing and savvy enough to spelunk the DOM to get it back.)

**Comment by markhalliwell** [comment]
Cross referencing: #74 #117
Possible solution: #151

**Comment by alexpott** [comment]
Here are reliable steps to create this problem:
1. Click the review button on a patch
2. Select a line of code to comment on
3. Enter comment
4. De-select the same line of code (before pressing save)
5. Press save
6. Press paste

**Comment by star-szr** [comment]
I can reproduce that, thanks a million @alexpott. I'll try and debug this.

**Comment by markhalliwell** [comment]
This fix is the `1.2.13` release.
